drivers/stmpe811/stmpe811.c

```diff
@@ -339,6 +358,14 @@ int stmpe811_read_touch_position(stmpe811_t *dev, stmpe811_touch_position_t *pos
     }
 #endif
 
+    /* Reset the FIFO, otherwise new touch data will be processed with a delay
+     * if the rate of calling this function to read the FIFO is slower than
+     * the rate at which the FIFO is filled. The reason for this is that with
+     * each call of this function always only the oldest touch data is read
+     * value by value from the FIFO. Gestures, for example, can't be
```

Comment:
This sentence is weird. Maybe there's an extra "always" ?

```suggestion
     * the rate at which the FIFO is filled. The reason for this is that with
     * each call of this function only the oldest touch data is read
     * value by value from the FIFO. Gestures, for example, can't be
```